node/runtime/pangolin/src/pallets/scheduler.rs

```diff
@@ -3,7 +3,7 @@ use frame_support::{traits::EqualPrivilegeOnly, weights::Weight};
 use pallet_scheduler::Config;
 use sp_runtime::Perbill;
 // --- darwinia-network ---
-use crate::{weights::pallet_scheduler::WeightInfo, *};
+use crate::*;
```

Comment:
leave this for later usage

Reply:
I did not get your meaning.  Do you want me to evert the change here? There are two compiler warnings here,

Reply:
yes, after the reviewing I'll add the benchmarks

Reply:
Okay. Let me push a revert commit